conga-aem-definitions/src/main/roles/aem-cms.yaml

```diff
@@ -191,12 +191,14 @@ config:
       flushTarget:
         #name: dispatcherAuthor1
         #url: http://localhost:8000
+        aliasUpdate: false
       # generate replicationagents package by default
       generatePackage: true
     publish:
       flushTarget:
         #name: dispatcherPublish1
         #url: http://localhost:8000
+        aliasUpdate: false
```

Comment:
please also add a comment to the parameter (and above) - the name of the parameter is not self-explanatory for those that are not fully informed about all features of the replication agents.

Reply:
done